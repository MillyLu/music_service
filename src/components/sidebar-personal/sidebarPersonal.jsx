import { useContext } from "react";
import { ThemeContext } from "../../ThemeProvider";
import { UserName } from "../sidebar-userName/userName";
import { UserAvatar } from "../sidebar-userAvatar/userAvatar";
import * as Styled from './styles';


export function SidebarPersonal() {
    const {theme} = useContext(ThemeContext);
    return(
        <Styled.SidebarPersonal theme={theme}>
            <UserName />
            <UserAvatar />
        </Styled.SidebarPersonal>
    )
}