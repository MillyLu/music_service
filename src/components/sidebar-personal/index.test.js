/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
// import { ShallowRenderer, createRenderer } from 'react-shallow-renderer';
import { SidebarPersonal } from './sidebarPersonal';



describe('<SidebarPersonal/>', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<SidebarPersonal />);
            const result = renderer.getRenderOutput();
            expect(result).toMatchSnapshot();
          })
      });