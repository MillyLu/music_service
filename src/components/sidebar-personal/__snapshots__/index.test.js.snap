// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SidebarPersonal/> should render successfully and match snapshot 1`] = `
<Styled(styled.div)
  theme={0}
>
  <UserName />
  <UserAvatar />
</Styled(styled.div)>
`;
